locking/locks.go

```diff
@@ -67,6 +67,8 @@ func NewClient(remote string, lfsClient *lfsapi.Client, cfg *config.Configuratio
 		cache:              &nilLockCacher{},
 		cfg:                cfg,
 		ModifyIgnoredFiles: lfsClient.GitEnv().Bool("lfs.lockignoredfiles", false),
+		LocalWorkingDir:    cfg.LocalWorkingDir(),
+		LocalGitDir:        cfg.LocalGitDir(),
```

Comment:
Do we make use of `LocalGitDir` at all?  If not, perhaps we can drop it here, and add it later should it become needed in a future change.

Big :+1: to the use of `cfg.LocalWorkingDir()` here because it validates all the comments I made about the preceding commit, I think!

Reply:
It's initialized elsewhere, and I think it's better to initialize them both at the same time, since it makes it easier to use them both together.